test/specs/bar.spec.js

```diff
@@ -72,6 +72,20 @@ define(['d3', 'bar', 'barChartDataBuilder'], function(d3, chart, dataBuilder) {
                 expect(actual).toBe(expected);
             });
 
+            it('should update color', () => {
+                let previous = barChart.colorSchema(),
+                    expected = '#FFFFFF',
+                    actual;
+
+                barChart.colorSchema([expected]);
+
+                const barColor = containerFixture.select('rect.bar');
+
+                containerFixture.call(barChart);
+
```

Comment:
Nit: We would do here:
  actual = barColor.attr('fill');

  expect(actual).toBe(expected);